Paper.tex

```diff
@@ -426,7 +426,7 @@ \subsubsection{Transaction Receipt}\linkdest{Transaction_Receipt}{}
 M_{3:2048}(\mathbf{x}: \mathbf{x} \in \mathbb{B}) & \equiv & \mathbf{y}: \mathbf{y} \in \mathbb{B}_{256} \quad \text{where:}\\
 \mathbf{y} & = & (0, 0, ..., 0) \quad \text{except:}\\
 \forall i \in \{0, 2, 4\}&:&\mathcal{B}_{m(\mathbf{x}, i)}(\mathbf{y}) = 1\\
-m(\mathbf{x}, i) &\equiv& \mathtt{KEC}(\mathbf{x})[i, i + 1] \bmod 2048
+m(\mathbf{x}, i) &\equiv& 2047 - (\mathtt{KEC}(\mathbf{x})[i, i + 1] \bmod 2048)
 \end{eqnarray}
 
 where $\mathcal{B}$ is the bit reference function such that $\mathcal{B}_{\mathrm{j}}(\mathbf{x})$ equals the bit of index $j$ (indexed from 0) in the byte array $\mathbf{x}$.
```

Comment:
I'd add a clarification here:
Within a byte $\mathcal{B}$ treats bits as big-endian (more significant bits have smaller indices).

Reply:
I agree! I've added that clarification, but for the entire byte-array